x-pack/plugin/eql/qa/common/src/test/java/org/elasticsearch/test/eql/EqlSpecLoaderTests.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.test.eql;
+
+import org.elasticsearch.test.ESTestCase;
+
+import java.util.List;
+
+public class EqlSpecLoaderTests extends ESTestCase {
```

Comment:
I think this test is not really needed. The description to be added to test_sample.toml and the tests in samples should be enough.